app/subscriber/src/features/my-reports/view/components/ReportEditForm.tsx

```diff
@@ -125,15 +125,6 @@ export const ReportEditForm: React.FC<IReportEditFormProps> = ({
   return (
     <PageSection
       tabIndex={0}
-      onKeyDown={(e) => {
```

Comment:
we don't need this anymore as the window listener will pick up for events whenever the edit form is mounted